lib/src/cluster_manager.dart

```diff
@@ -105,30 +137,71 @@ class ClusterManager<T extends ClusterItem> {
   }
 
   /// Method called on camera move
-  void onCameraMove(CameraPosition position, {bool forceUpdate = false}) {
+  void onCameraMove(CameraPosition position, {bool forceUpdate = false, bool allowSkip = false}) {
     _zoom = position.zoom;
     if (forceUpdate) {
-      updateMap();
+      updateMap(allowSkip: allowSkip);
     }
   }
 
-  /// Retrieve cluster markers
-  Future<List<Cluster<T>>> getMarkers() async {
+  Future<LatLngBounds> _addPadding(LatLngBounds mapBounds) async {
+    final northEastL = mapBounds.northeast;
+    final southWestL = mapBounds.southwest;
+
+    if (padding != null) {
+      final [northEastC, southWestC] = await Future.wait([
+        GoogleMapsFlutterPlatform.instance
+            .getScreenCoordinate(northEastL, mapId: _mapId!),
+        GoogleMapsFlutterPlatform.instance
+            .getScreenCoordinate(southWestL, mapId: _mapId!),
+      ]);
+
+      final [northEastP, southWestP] = await Future.wait([
+        GoogleMapsFlutterPlatform.instance.getLatLng(
+          northEastC.add(
+            x: padding!.right.toInt(),
+            y: -padding!.top.toInt(),
+          ),
+          mapId: _mapId!,
+        ),
+        GoogleMapsFlutterPlatform.instance.getLatLng(
+          southWestC.add(
+            x: -padding!.left.toInt(),
+            y: padding!.bottom.toInt(),
+          ),
+          mapId: _mapId!,
+        ),
+      ]);
+      return LatLngBounds(southwest: southWestP, northeast: northEastP);
+    }
+    return LatLngBounds(southwest: southWestL, northeast: northEastL);
+  }
+
+  Future<bool> shallUpdate(List<T> currentVisible) async {
```

Comment:
Do you really need a Future here?

Reply:
Yes, you're absolutely right, I had overlooked that. However, I pushed the commit to the wrong branch, as it doesn't belong to the padding stuff at all. I have now removed the commit.